go/vt/vtgate/executor.go

```diff
@@ -991,6 +994,119 @@ func (e *Executor) showTablets(show *sqlparser.ShowLegacy) (*sqltypes.Result, er
 	}, nil
 }
 
+func (e *Executor) showVitessReplicationStatus(ctx context.Context, show *sqlparser.ShowLegacy) (*sqltypes.Result, error) {
+	ctx, cancel := context.WithTimeout(ctx, *HealthCheckTimeout)
+	defer cancel()
+	rows := [][]sqltypes.Value{}
+
+	// This is only used for tests
+	if UsingLegacyGateway() {
+		status := e.scatterConn.GetLegacyHealthCheckCacheStatus()
+
+		for _, s := range status {
+			for _, ts := range s.TabletsStats {
+				if ts.Tablet.Type != topodatapb.TabletType_REPLICA {
```

Comment:
Is it intentional to limit this to `REPLICA` types? `RDONLY` tablets also replicate from primary.

Reply:
It was intentional but I wasn’t sure if it was the right thing to do either. 🙂 Would you prefer rdonly too with another column showing the tablet type?

Reply:
Yes I think that will be helpful.

Reply:
OK, thanks! I made this change.